README.md

```diff
@@ -67,7 +67,7 @@ sudo apt-get install \
 Additionally, you need the [Haskell Tool
 Stack](https://docs.haskellstack.org/en/stable/README/). If your distro isn't
 shipping a new enough version of it, you may encounter some non-obvious build
-failures, so we recommend installing the latest Stack outside of package
+failures, so we recommend installing the latest Stack outside of the package
```

Comment:
To my ear, this reads better with out the additional "the". Can you please revert its addition?

Reply:
sure, let me do that. But I need to get this merged for hacktoberfest. This pull will be counted in the challenge.